evennia/commands/default/general.py

```diff
@@ -424,8 +425,9 @@ def func(self):
         if not success:
             caller.msg("This can't be picked up.")
         else:
-            caller.msg(f"You pick up {obj.name}.")
-            caller.location.msg_contents(f"{caller.name} picks up {obj.name}.", exclude=caller)
+            singular, _ = obj.get_numbered_name(1, caller)
+            caller.msg(f"You pick up {singular}.")
+            caller.location.msg_contents(f"{caller.name} picks up {singular}.", exclude=caller)
```

Comment:
While we are at updating this string, we could update this for the modern way to express this using funcparser embeds:

```python
caller.location.msg_contents(f"$You() $conj(pick) up {singular}.", from_obj=caller)
```
With this, there is no need for a separate `callerm.msg()` on the line above.